src/ansys/hps/client/auth/api/auth_api.py

```diff
@@ -164,26 +168,28 @@ def delete_user(self, user: User) -> None:
         return self.keycloak_admin_client.delete_user(user.id)
 
 
-def _admin_client(client):
+def _admin_client(client: "Client"):
     """Set information for admin.
```

Comment:
Descriptions should be in plain English. Spell out "admin" to "administrator" if that is what is meant.